src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -694,7 +773,7 @@ FixedwingPositionControl::control_position(const hrt_abstime &now, const Vector2
 						   false,
 						   radians(_param_fw_p_lim_min.get()));
 
-		} else if (pos_sp_curr.type == position_setpoint_s::SETPOINT_TYPE_LOITER) {
+		} else if (position_sp_type == position_setpoint_s::SETPOINT_TYPE_LOITER) {
```

Comment:
@dagar This is a bit confusing. Above you are using `pos_sp_curr.type` while here you are using `position_sp_type`

Reply:
No real reason, nasty rebase.